eng/Versions.props

```diff
@@ -225,7 +225,7 @@
     <CastleCorePackageVersion>4.2.1</CastleCorePackageVersion>
     <FSharpCorePackageVersion>4.2.1</FSharpCorePackageVersion>
     <GoogleProtobufPackageVersion>3.8.0</GoogleProtobufPackageVersion>
```

Comment:
We should update this as well. The latest version is 3.9.1. Not blocking though.

Reply:
Grpc.AspNetCore references Google.Protobuf itself.

This reference is only used by SignalR demos of using protobuf as the serializer so I didn't change it.